spec/support/datatables_shared_examples.rb

```diff
@@ -143,3 +141,67 @@
     end
   end
 end
+
+# let(:new_content) { "#[Title]#\nTitle\n\n#[New Field]#\nNew Field Value" }
+# let(:old_content) { "#[Title]#\nTitle" }
+# let(:resource) { Issue.take }
+# let(:resource_attribute) { 'text' }
+shared_examples 'a DataTable with Dynamic Columns' do
+  let(:hide_default_columns) do
+    within '.dt-buttons.btn-group' do
+      page.find('.buttons-colvis').click
+
+      within '.dt-button-collection' do
+        click_link 'Created'
+        click_link 'Updated'
+      end
+    end
+  end
+
+  let(:update_resource_with_new_content) do
+    attributes = Hash[resource_attribute, new_content]
+    resource.update(attributes)
```

Comment:
Would something like this work instead:

```suggestion
    resource.update_attribute(resource_attribute, new_content)
```